xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
> FWIW, if literal tabs are felt to be problematic, then $chr{9} can be used.

That's neat. I'd like to go with this if we can finally agree upon using tab-delimited values?

> Why is CSV-style quoting useful when we're using a tab separator? Is it essentially that we need to quote the string if it contains literal tabs, and $csv in "always-quote" mode achieves that?

Yes, query strings might contain tabs in which case it becomes essential to use quoting. I also checked the final log file entries with and without intentional tabs in query strings and I found that entries with tabs in queries leads were generally erroneous.